File: 2017/Python/J2.py
N = int(input())
K = int(input())

"""
sum = 0 #SOLUTION 1
for i in range (K+1):
    sum+=(N*10**i)
print (sum)
"""

print(sum([N*10**i for i in range (K+1)])) #SOLUTION 2

File: 2017/Java/J2.java
import java.util.*;
public class Main {
	public static void main(String args[]) {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        int k = input.nextInt();
        double sum = 0;
        for (int i = 0; i<k+1; i++){
            sum+=n*Math.pow(10,i);
        }
        System.out.println((int)sum);
    }
}
